packages/lodestar/src/network/nodejs/util.ts

```diff
@@ -22,3 +27,16 @@ export async function loadPeerId(path: string): Promise<PeerId> {
   const data = await promisify(fs.readFile)(path);
   return await promisify(PeerId.createFromJSON)(JSON.parse(data));
 }
+
+/**
+ * 
+ * @param peerId Create an instance of NodejsNode asynchronously
+ */
+export async function createLibP2p(peerIdOrPromise: PeerId | Promise<PeerId>, 
```

Comment:
:+1:

Reply:
I've modified network a bit in PR to this branch, there is a link in eth2 channel as I can't tag you on his repo